frontend/src/utils/fetch-mock.js

```diff
@@ -0,0 +1,27 @@
+import {apiHost} from './networkConfig';
+
+const model = {
+	id : 11,
+	user_id : 22,
+	project_id : 44,
+	hyperparameters : {a:"b"},
```

Comment:
That's the spirit!

Reply:
That was a mistake on my part, I'm terribly sorry for that